### PR TITLE
FP 0.18 Implement WhatsApp parsing

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Parse raw report and ignore checks from probe due to https://github.com/ooni/probe-engine/issues/341

### CODE REVIEW

File: af/fastpath/fastpath/utils.py [hunk 1]
```diff
@@ -498,7 +498,7 @@
         {
             "body_match": "<iframe src=\"http://subblock.mts.ru/api",
             "locality": "isp"
-        }
+        },
```
Comment (hellais) [inline af/fastpath/fastpath/utils.py:501]: 👍 apologies for the typos, should  probably have run it through a linter 😬

**Review by hellais — APPROVED** [review]
🐳

File: af/fastpath/fastpath/core.py [hunk 2]
```diff
@@ -494,6 +494,26 @@ def score_measurement_facebook_messenger(msm):
     return scores
 
 
+def _extract_tcp_connect(tk):
+    # https://github.com/ooni/spec/blob/master/data-formats/df-005-tcpconnect.md
+    # NOTE: this is *NOT* ts-008-tcp-connect.md
```
Comment (bassosimone) [inline af/fastpath/fastpath/core.py:499]: Yeah, I know, this is confusing :-(

Comment (bassosimone) [reply af/fastpath/fastpath/core.py:499]: Since we can rename test helpers without too much pain, maybe `df-005-tcpdial.md`?

**Review by bassosimone — APPROVED** [review]
🐳 looks good to me

File: af/fastpath/fastpath/core.py (hunk 2 above)
Comment (hellais) [reply af/fastpath/fastpath/core.py:499]: We should eventually relocate all legacy tests to some special tree or mark them as deprecated in a more obvious way. I would not worry to much about the naming clash as the key names are in fact identical `test_name: tcp_connect` and `tcp_connect: []`.

I do agree it's confusing though :P

Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py:499]: The tricky part is in the key names indeed. Let's just keep the comments in the code as reminders.

**Pull request merged**